gensim/test/test_similarities.py

```diff
@@ -257,6 +270,47 @@ class TestMatrixSimilarity(unittest.TestCase, _TestSimilarityABC):
     def setUp(self):
         self.cls = similarities.MatrixSimilarity
 
+class TestWmdSimilarity(unittest.TestCase, _TestSimilarityABC):
+    def setUp(self):
+        self.cls = similarities.WmdSimilarity
+        self.w2v_model = Word2Vec(texts, min_count=1)
+
+    def testFull(self, num_best=None):
+        # Override testFull.
+        index = self.cls(texts, self.w2v_model)
+        index.num_best = num_best
+        query = texts[0]
+        sims = index[query]
+    
+        if num_best is not None:
+            # Sparse array.
+            self.assertTrue(numpy.alltrue(sims != 0.0))
+        else:
+            self.assertTrue(sims[0] == 0.0)  # Similarity of a document with itself is 0.0.
+            self.assertTrue(numpy.alltrue(sims[1:] != 0.0))
```

Comment:
@tmylk maybe `_TestSimilarityABC` (which I just extend to WMD) should contain these `num_best` tests? Specifically in `testNumBest`.